src/Orleans/AssemblyLoader/AssemblyLoader.cs

```diff
@@ -291,15 +81,15 @@ private static Assembly MatchWithLoadedAssembly(AssemblyName searchFor, IEnumera
                 var candidateFullName = assembly.FullName;
                 if (String.Equals(candidateFullName, searchForFullName, StringComparison.OrdinalIgnoreCase))
                 {
-                    return assembly;                    
+                    return assembly;
                 }
             }
             return null;
         }
 
         private static Assembly MatchWithLoadedAssembly(AssemblyName searchFor, AppDomain appDomain)
         {
-            return 
+            return
```

Comment:
hehe, funny that this was mentioned, but yes, until Assembly loading gets fixed, I'm introducing a AppDomain shim for now, otherwise there is no way I can get help fixing all the porting holes we currently have, and would have to spend a lot of time fixing every build break in the correct manner myself until I can get help with it. A PR is coming *really* soon.